jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -119,15 +119,24 @@ def _parse_hdu(app, hdulist, file_name=None):
         if hdu.data.dtype in (int, np.uint, np.uint32) or \
                 any(x in hdu.name.lower() for x in EXT_TYPES['mask']):
             app.add_data_to_viewer('mask-viewer', data_label)
+            viewer = app.get_viewer('mask-viewer')
+            viewer.state.x_att_world = app.data_collection[data_label].id['Right Ascension']
```

Comment:
Does this work with data with no WCS? I am trying to think how this would fit into #1040 .

Reply:
I think I need a case here to fall back on `Pixel Axis 0 [z]` (or whichever is appropriate) for things with no WCS.

Reply:
I am not sure which axis that would be. I have found that specutils only attempt to swapaxes if it detects a WCS, so for anything without WCS, I needed to transpose. If you look at PR 1040, you will see a lot of `array.T` happening here and there. 🤪  Do I still need that with this patch?